Makefile

```diff
@@ -80,13 +80,16 @@ man: bin/docs ## update manpages
 	@echo "Update generated manpages"
 	@./bin/docs --target=./docs/man/man1 --kind=man
 
+.PHONY: generated
+generated: test-unit-update-golden man docs fmt ## generate all files that needs to be generated
```

Comment:
I am a little bit worried to put `test-unit-update-golden` in `generated` as this might update golden file when not intended by the user (and thus hide a unwanted change in the output)

Reply:
why would it be? this should not happen isnt it ? I mean the point is to have the same exact output unless a code change?

Reply:
@vdemeester ping!

Reply:
lol, not sure :stuck_out_tongue: